configure.ac

```diff
@@ -927,6 +927,12 @@ AS_IF([test "x$enable_slingshot" = xyes],[
         [AC_MSG_ERROR([libcxi or its headers not found])])
 ])
 
+AC_ARG_ENABLE([slingshot_switch],
+    [AS_HELP_STRING([--enable-slingshot_switch], [require the slinghost on-switch plugins @<:@default=check@:>@])],
+    [],
+    [enable_slingshot_switch="check"])
```

Comment:
While it won't actually change the behavior, it would be better to have this say enable_slingshot_switch="yes", because there is no actually checking of dependencies. If build is not explicitly disabled it _will_ attempt to build it.